rst2pdf/createpdf.py

```diff
@@ -94,7 +94,7 @@
 from rst2pdf.image import MyImage, missing
 from rst2pdf.aafigure_directive import Aanode
 from rst2pdf.log import log, nodeid
-from rst2pdf.smartypants import smartyPants
+import smartypants
```

Comment:
Shouldn't we do `from smartypants import smartypants` as per `basenodehandler.py`?

Reply:
It's the same thing, but sure, why not.